migration/src/m20230907_175501_add_aggregator_onboarding_boolean_to_account.rs

```diff
@@ -0,0 +1,58 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, db: &SchemaManager) -> Result<(), DbErr> {
+        db.alter_table(
+            TableAlterStatement::new()
+                .table(Account::Table)
+                .add_column(
+                    ColumnDef::new(Account::IntendsToUseSharedAggregators)
+                        .boolean()
+                        .not_null()
+                        .default(false),
+                )
+                .to_owned(),
+        )
+        .await?;
+
+        db.get_connection()
+            .execute_unprepared(
+                "UPDATE account
```

Comment:
Suggestions on improving this query are very welcome. It's a bit awkward but performance is irrelevant here